src/components/Radio/index.tsx

```diff
@@ -0,0 +1,63 @@
+import React from "react";
+import styles from "./styles.module.scss";
+import variants from "./variants.module.scss";
+import "./variants.module.scss";
+import { concatStyles } from "@/utils/concatStyles";
+import { Icon } from "../Icon";
+
+export type RadioVariant = "default" | "success";
+
+export interface RadioOption {
+  label: string;
+  value: string;
+}
+
+export interface RadioProps {
+  options: RadioOption[];
+  value?: string;
+  onSelect: (value: string) => void;
+  variant?: RadioVariant;
+  className?: string;
+}
+
+export const Radio: React.FC<RadioProps> = ({
+  options,
+  value,
+  onSelect,
+  variant = "default",
+  className,
+}) => {
```

Comment:
É bom mantermos o padrão 
```suggestion
export const Radio = ({
  options,
  value,
  onSelect,
  variant = "default",
  className,
}:RadioProps) => {
```